src/interfaces/wallet.cpp

```diff
@@ -274,8 +274,13 @@ class WalletImpl : public Wallet
         CAmount& new_fee,
         CMutableTransaction& mtx) override
     {
-        return feebumper::CreateTransaction(m_wallet.get(), txid, coin_control, total_fee, errors, old_fee, new_fee, mtx) ==
-               feebumper::Result::OK;
+        if (total_fee > 0) {
+            return feebumper::CreateTotalBumpTransaction(m_wallet.get(), txid, coin_control, total_fee, errors, old_fee, new_fee, mtx) ==
```

Comment:
Could have this in a function:
```
feebumper::CreateTransaction(...)
    if (total_fee > 0) {
        return feebumper::CreateTotalBumpTransaction(...) 
    } else {
        return feebumper::CreateRateBumpTransaction(...)
    }
}
```

Reply:
`CreateTransaction` always messed with my ctags anyways because of CWallet's ... :P